app/models/planning_application.rb

```diff
@@ -337,6 +338,21 @@ def secure_change_url
     public_url_enabled? ? "#{protocol}://#{public_url}" : "#{protocol}://#{internal_url}"
   end
 
+  def site_notice_link
+    protocol = Rails.env.production? ? "https" : "http"
+    path = "planning_applications/#{id}/site_notices/download"
+
+    if Bops.env.production?
+      "#{protocol}://planningapplications.#{local_authority.subdomain}.gov.uk/#{path}"
+    else
+      "#{protocol}://#{local_authority.subdomain}.bops-applicants.services/#{path}"
+    end
+  end
```

Comment:
Would it be worth having a variable somewhere for the bops-applicants URL, to ensure we're being consistent about it? I know there's an `APPLICANTS_APP_HOST` variable that's used in a couple of places but not sure if that relates to this (or should).